examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3685,7 +4006,15 @@ class GLTFParser {
 			const input = animationDef.parameters !== undefined ? animationDef.parameters[ sampler.input ] : sampler.input;
 			const output = animationDef.parameters !== undefined ? animationDef.parameters[ sampler.output ] : sampler.output;
 
-			pendingNodes.push( this.getDependency( 'node', name ) );
+			const useExtension = target.extensions && target.extensions[ EXTENSIONS.KHR_ANIMATION_POINTER ];
+
+			let nodeDependency = undefined;
+			if ( useExtension )
+				nodeDependency = this.getAnimationPointerDependency( target );
+			else
+				nodeDependency = this.getDependency( 'node', name );
```

Comment:
One of the larger hurdles on this extension will be that glTF definitions do not translate 1:1 into three.js objects —

- Materials will be cloned depending on the context in which they're used. The same glTF material applied to (1) points, (2) lines, (3) mesh with vertex colors, (4) mesh with vertex normals, (5) mesh with vertex tangents, will result in 5x copies of the material. 
- Lights and Cameras will be cloned if they're attached at >1 Node.
- Probably similar for KHR_audio if that gets ratified?

Reply:
Do you have an idea on how to tackle these in the context of this extension? I'd assume that when they are split, the correct additional tracks would need to be created.

One idea would be keeping some `importUUID` around, which could then be used during `findNode` to map back to objects.